feat(core)!: Implement floating UI for tooltip positioning and styling (#108)
stackbuilders/react-native-spotlight-tour: README.md

[![GitHub Release Date](https://img.shields.io/github/release-date/stackbuilders/react-native-spotlight-tour)](https://github.com/stackbuilders/react-native-spotlight-tour/releases)
[![Snyk Vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/@stackbuilders/react-native-spotlight-tour)](https://snyk.io/)

> `react-native-spotlight-tour` is a simple and intuitive library for React Native (Android, iOS, and Web

Reviewer: Thanks for the corrections. Maybe we should also explain that the library uses `floting-ui/react-native` and re-exports all their middlewares so they can be used on the configuration, wdyt? 🙂

[![GitHub Release Date](https://img.shields.io/github/release-date/stackbuilders/react-native-spotlight-tour)](https://github.com/stackbuilders/react-native-spotlight-tour/releases)
[![Snyk Vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/@stackbuilders/react-native-spotlight-tour)](https://snyk.io/)

`react-native-spotlight-tour` is a simple and intuitive library for React Native (Android, iOS, and Web
compatible). It uses [Floating UI](https://floating-ui.com) under the hood in order to handle elements

> positioning, it re-exports all floating ui middlewares so they can be configured in the tour.

Reviewer: ```suggestion
positioning, it re-exports all floating-ui middlewares to be configured in the tour.
```
